src/frontend/web_application/src/modules/draftMessage/components/ReplyForm/presenter.jsx

```diff
@@ -99,94 +101,102 @@ class ReplyForm extends Component {
   render() {
     const {
       user, parentMessage, renderDraftMessageActionsContainer, isSending, renderAttachments,
-      draftFormRef,
+      draftFormRef, lastMessage,
     } = this.props;
     const dropdownId = uuidV1();
     const excerpt = parentMessage && parentMessage.excerpt;
 
-    return (
-      <DiscussionDraft className="m-reply" draftFormRef={draftFormRef}>
-        <div className="m-reply__avatar-col">
-          <ContactAvatarLetter
-            contact={user.contact}
-            className="m-reply__avatar"
-          />
-        </div>
-        <form method="POST" className="m-reply__form-col">
-          <TopRow className="m-reply__top-bar">
-            <div className="m-reply__top-bar-info">
-              <div className="m-reply__author"><Trans id="reply-form.you">You</Trans></div>
-              {this.state.draft.type && this.renderDraftType()}
-              <div className="m-reply__date"><Trans id="reply-form.now">Now</Trans></div>
-            </div>
-
-            <DropdownControl toggleId={dropdownId} className="m-reply__top-actions-switcher" icon="ellipsis-v" />
-
-            <Dropdown
-              id={dropdownId}
-              className="m-reply__top-actions-menu"
-              alignRight
-              isMenu
-              closeOnClick="all"
-            >
-              {renderDraftMessageActionsContainer()}
-            </Dropdown>
-          </TopRow>
-          {parentMessage && (
-            <div className="m-reply__parent">
-              <Link to={`#${parentMessage.message_id}`} className="m-reply__parent-link">
-                <Trans id="reply-form.in-reply-to">In reply to: {excerpt}</Trans>
-              </Link>
-            </div>
-          )}
-          <BodyRow className="m-reply__body">
-            <DiscussionTextarea
-              body={this.state.draft.body}
-              onChange={this.handleChange}
+    if ((parentMessage && parentMessage.type === 'email') || (lastMessage && lastMessage.type === 'email')) {
```

Comment:
It's convenient that Twitter messages does not have a reply button ^^

Reply:
yes :)